source/text.c

```diff
@@ -109,6 +126,8 @@ void C2D_TextBufDelete(C2D_TextBuf buf)
 void C2D_TextBufClear(C2D_TextBuf buf)
 {
 	buf->glyphCount = 0;
+	for (C2Di_Glyph* g = buf->glyphs; g != buf->glyphs + buf->glyphBufSize; g++)
```

Comment:
I sense a C++ism here. Wouldn't it be better to just memset the whole thing to zero? `buf->glyphBufSize*sizeof(C2Di_Glyph)`

Reply:
I figured that would actually take more time. Would you rather it just memset the whole thing to zero?

Reply:
What you're doing here is basically random memory access with bitfields (i.e. read/modify/write). I think a memset (with clean sequential memory accesses) may actually be faster than that.

Reply:
Removed that field; it had planned usage that was never actually realized